scripts/dev/infra-sync

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+# Fetches cluster artifacts from infra.rox.systems and regenerates your ~/.kube/config.
+
+set -euo pipefail
+
+infra_dir="$HOME/.kube/infra"
+clusters_dir="$infra_dir/clusters"
+
+os=$(uname -o)
+OS_MACOS="Darwin"
+
+export INFRA_TOKEN="${INFRA_TOKEN:-}"
+if [[ "${INFRA_TOKEN}" == "" ]]; then
+    if [[ "${os}" == "Darwin" ]]; then
+        case "${os}" in
+            "${OS_MACOS}")
+                if token=$(security find-generic-password -s infra.rox.systems -D token -a "" -l acs -w); then
+                    INFRA_TOKEN="${token}"
+                else
+                    echo >&2 "Failed to retrieve token from Keychain"
+                    echo >&2 "Use the following command for saving your infra token in Keychain:"
+                    echo >&2
+                    echo >&2 "  $ security add-generic-password -s infra.rox.systems -D token -a \"\" -l acs -T \"\" -w \"YOUR_INFRA_TOKEN\""
+                    echo >&2
+                    exit 1
+                fi
+                ;;
+        esac
+
+    fi
+
+fi
+
+mkdir -p "${clusters_dir}"
+
+echo "Syncing your infra clusters..."
+
+for cluster_dir in "${clusters_dir}"/*; do
+    cluster_name=$(basename "${cluster_dir}")   
+    rm -rf "${clusters_dir:?}/$cluster_name"
+done
```

Comment:
The `rm -rf ${clusters_dir}/*` call fails in case the directory '${clusters_dir}` does not contain any entries.
But, I just realized that also my approach had a problem:
If `${clusters_dir}` is empty, then my original for-loop will try to delete the single directory `${clusters_dir}/*`, i.e. with a literal asterisk as name.
I have replaced it with a hopefully solid version using `find`.

Reply:
... including additional safety checks.

Reply:
Ok to resolve, @tommartensen?